Tests/FetchingLogictests/RequestBuilderTests.swift

```diff
@@ -331,6 +333,63 @@ class RequestBuilderTests: XCTestCase, CLLocationManagerDelegate {
 
     }
     
+    //MARK: - Targeting External UserIds UserDefault
+    func testPostDataWithTargetingExternalUserIds() throws {
```

Comment:
Please add a test for ExternalUserId, where it is initialised with source = identifier = atype = ext = nil